src/assets/i18n/ru.json

```diff
@@ -25,9 +25,26 @@
     "CHOOSE_LANG": "Выбрать язык"
   },
   "INFO": {
-    "MANUAL": "Руководство",
+    "MANUAL": {
+      "TITLE": "Руководство",
+      "DESCRIPTION": "Описание проєкта"
```

Comment:
Please, take the project description from the `readme.md` file of the master branch and add it in other languages